docs/hugo/content/introduction/multitenant-deployment.md

```diff
@@ -0,0 +1,68 @@
+To deploy the operator in multi-tenant mode the release YAML has been split into two parts:
+
+1. Cluster-wide resources:
+   * Custom resource definitions for the Azure resources
+   * Cluster roles for managing those resources
+   * The `azureserviceoperator-system` namespace containing the deployment and service to run the operator.
+     The webhook deployment of the operator is configured to run in webhook-only mode.
+     It doesn't require any Azure credentials, since this operator never talks to ARM, only to the kubernetes API server.
+   * Webhook configuration referring to that service.
+
+2. Per-tenant resources:
+   * A namespace containing the deployment to run the tenant operator, configured for watchers-only mode.
+   * The `aso-controller-settings` secret defining the Azure credentials that should be used, and the set of namespaces that this operator will watch for Azure resources.
+   * A cluster role binding enabling the per-tenant operator's service account to manage the Azure resources.
+
+## Example files
+Examples of the deployment YAML files are available on the release page for ASO v2 releases from [v2.0.0-alpha.5](https://github.com/Azure/azure-service-operator/releases/tag/v2.0.0-alpha.5).
+The cluster-wide file `multitenant-cluster_v2.0.0-alpha.5.yaml` can be used as-is (the webhook operator namespace is fixed as `azureserviceoperator-system`),
+but the namespaces and cluster role binding in the per-tenant file `multitenant-tenant_v2.0.0-alpha.5.yaml` will need to be updated from `tenant1` to the desired name for each tenant.
+
+## Per-tenant configuration
+Create the `aso-controller-settings` secret as described in the [authentication docs](https://azure.github.io/azure-service-operator/introduction/authentication/),
+but create the secret in the tenant namespace and add an extra target namespaces key to it:
+```
+export TENANT_NAMESPACE="<tenant namespace>"
+export AZURE_SUBSCRIPTION_ID="<subscription id>"
+export AZURE_TENANT_ID="<tenant id>"
+export AZURE_CLIENT_ID="<client id>"
+export AZURE_CLIENT_SECRET="<client secret>"
+export AZURE_TARGET_NAMESPACES="<comma-separated namespace names>"
+kubectl create namespace "$TENANT_NAMESPACE"
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: aso-controller-settings
+  namespace: $TENANT_NAMESPACE
+stringData:
+  AZURE_SUBSCRIPTION_ID: "$AZURE_SUBSCRIPTION_ID"
+  AZURE_TENANT_ID: "$AZURE_TENANT_ID"
+  AZURE_CLIENT_ID: "$AZURE_CLIENT_ID"
+  AZURE_CLIENT_SECRET: "$AZURE_CLIENT_SECRET"
+  AZURE_TARGET_NAMESPACES: "$AZURE_TARGET_NAMESPACES"
+EOF
+```
+
+## Role handling
+The multi-tenant deployment example files have a cluster role granting access to the Azure resource types whichever namespaces they are created in,
```

Comment:
Phrasing? This part reads weird to me:
"have a cluster role granting access to the Azure resource types whichever namespaces they are created in"

Reply:
Yeah, reads like a partial edit... 
Ok, I've updated the wording for that section.